### PR TITLE
Product rename

### PR DESCRIPTION
**Pull request by jglick**
@reviewbybees

### CODE REVIEW

**Comment by stephenc** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-Jenkins Enterprise by CloudBees Plugin
+CloudBees Jenkins Enterprise Plugin
 ======================================
 
-Read more: [https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+Enterprise+by+CloudBees](https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+Enterprise+by+CloudBees)
+Read more: https://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Jenkins+Enterprise
```
Comment (oleg-nenashev) [inline README.md:4]: Off-topic: Does our Confluence version support links to moved pages? There could be orphaned links in other places

**Comment by oleg-nenashev** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (jglick) [inline README.md:4]: > Does our Confluence version support links to moved pages?

According to [CONF-2906](https://jira.atlassian.com/browse/CONF-2906), no, yet the old page is still rendered, I think by the custom caching reverse proxy, but it cannot be _edited_, so I cannot set up a manual redirect. Sigh.

Comment (oleg-nenashev) [inline README.md:4]: I suppose nobody from the community will blame a stub page with "This product has been renamed to XXX [with some text and governance meeting minutes links]"

Comment (jglick) [inline README.md:4]: That stub page is exactly what I am unable to create, I think due to a buggy reverse proxy.

Comment (jglick) [inline README.md:4]: [INFRA-301](https://issues.jenkins-ci.org/browse/INFRA-301)

**Pull request merged**